pyanaconda/payload/rpmostreepayload.py

```diff
@@ -64,6 +64,14 @@ def spaceRequired(self):
         # We don't have this data with OSTree at the moment
         return Size("500 MB")
 
+    @property
+    def needsNetwork(self):
+        """Test ostree repository if it requires network."""
+        if self.data.ostreesetup.url and self.data.ostreesetup.url.startswith("file://"):
```

Comment:
Could just be:
`return not (self.data.ostreesetup.url and self.data.ostreesetup.url.startswith("file://"))`
but LGTM as is too!

Reply:
this is how pylint likes it, IIRC, yes :)

Reply:
I decided for if command because somehow it felt more readable to my weary brain, but here you are :)